From 28c50b8654fb363b926558bf23b5e6a638bd13d1 Mon Sep 17 00:00:00 2001
From: Quynh Nguyen <quynhnguyen.xb@renesas.com>
Date: Wed, 14 Oct 2020 16:58:26 +0700
Subject: [PATCH 320/475] dt-bindings: ata: sata_rcar: Add device tree support
 for r8a7743

Document SATA device tree support for the Renesas R8A7743 (RZ/G1M) SoC.

Signed-off-by: Quynh Nguyen <quynhnguyen.xb@renesas.com>
---
 Documentation/devicetree/bindings/ata/sata_rcar.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/ata/sata_rcar.txt b/Documentation/devicetree/bindings/ata/sata_rcar.txt
index a6bb9cf..0032cb9 100644
--- a/Documentation/devicetree/bindings/ata/sata_rcar.txt
+++ b/Documentation/devicetree/bindings/ata/sata_rcar.txt
@@ -2,6 +2,7 @@
 
 Required properties:
 - compatible		: should contain one of the following:
+			  - "renesas,sata-r8a7743" for the R8A7743 SoC
 			  - "renesas,sata-r8a7744" for the R8A7744 SoC
 			  - "renesas,sata-r8a7779" for R-Car H1
 			    ("renesas,rcar-sata" is deprecated)
-- 
2.7.4

